compiler/dcalc/ast.ml

```diff
@@ -154,11 +181,49 @@ let make_let_in (x : Var.t) (tau : typ Pos.marked) (e1 : expr Pos.marked Bindlib
     (e2 : expr Pos.marked Bindlib.box) (pos : Pos.t) : expr Pos.marked Bindlib.box =
   make_app (make_abs (Array.of_list [ x ]) e2 pos [ tau ] pos) [ e1 ] pos
 
-let make_multiple_let_in (xs : Var.t array) (taus : typ Pos.marked list)
-    (e1 : expr Pos.marked Bindlib.box list) (e2 : expr Pos.marked Bindlib.box) (pos : Pos.t) :
-    expr Pos.marked Bindlib.box =
-  make_app (make_abs xs e2 pos taus pos) e1 pos
```

Comment:
why did you removed this function ? It could still be useful (for instance lines 194 and following)